js/initcontext.go

```diff
@@ -113,15 +113,29 @@ func (i *InitContext) Require(arg string) goja.Value {
 		}
 		return v
 	default:
+		// Try in the plugin store
```

Comment:
So, here it is. I don't like the idea of plugins having their own dedicated prefix. We should check for `k6` prefixed things, and if a plugin doesn't expose a module, fall back to a file. I suspect that it could be expressed differently (read: much much better) in this code though.

Reply:
>  I don't like the idea of plugins having their own dedicated prefix.

After a first read, I agreed with you, but on second thought I think it might be beneficial for clarity (as in, where the code executed actually comes from) if it's isolated into its own namespace. Using something like `k6-plugin/*` would remove that cognitive overhead.

Reply:
Changed it to include the prefix. Thanks for the feedback!